include/struct_decls/struct_02026324_decl.h

```diff
@@ -1,6 +1,6 @@
 #ifndef POKEPLATINUM_STRUCT_02026324_DECL_H
 #define POKEPLATINUM_STRUCT_02026324_DECL_H
 
-typedef struct UnkStruct_02026324_t UnkStruct_02026324;
+typedef struct PokedexData UnkStruct_02026324;
```

Comment:
Label the typedef as well:

```suggestion
typedef struct PokedexData PokedexData;
```

Reply:
Done, I did this in two separate commits to make it easier to look over the new changes without getting assaulted by 200 file changes